### ISSUE TITLE
The toast couldn`t close manual

### ISSUE THREAD

**Issue by muyea** [noru/vue-easy-toast#22]
I implemented it in another way is display a new toast，then set the duration to zero.
there is the code:
```javascript
$vm.$toast(null, { duration: 0 })
```
is there a better way to do that?

**Comment by noru** [comment]
Currently no.

From UI control point of view, I think a 'toast' means to be there for the whole duration (not dismissable, per se). That's why it should stay unobtrusive and interaction unawareness. 

By having a 'dismiss()', would encourage people using it like this:

```javascript .  
toast.dismiss()  // in case there is a previous one .  
toast('something else') .  // or do other ui related things .  
``` .  

It worth considering though. Maybe developers should be able to use it at their own discretion. I would like to make it dismissable, but I would not recommend anyone to use it. And you should rethink your reason to 'undo' your first toast.

**Comment by muyea** [comment]
suppose such a scene:
I forward page B from page A, In page B, i did something make the toast display, 
then I back to page A,  the toast still lived there! This will make something wrong.

**Comment by derrickroccka** [comment]
I agree! I love the widget, it works like a charm, but I'd love to have the dismiss option available ;)

**Comment by rewkha** [comment]
@mrios Before noru's answer, you can just use git as source of dependency. Just add this into your **package.json** instead of regular defining:
`"vue-easy-toast": "git+https://github.com/noru/vue-easy-toast.git"`

**Comment by mrios** [comment]
Thanks @rewkha , I ran `npm install https://github.com/noru/vue-easy-toast.git` trying yesterday and finally worked.
